components/org.wso2.carbon.privacy.forgetme.tool/src/main/java/org/wso2/carbon/privacy/forgetme/ForgetMeTool.java

```diff
@@ -223,4 +232,35 @@ private static UserIdentifier createUserIdentifier(CommandLine cmd) throws Comma
         return userIdentifier;
     }
 
+    /**
+     * Add JAR files found in the given directory to the Classpath. This fix is done due to terminal's argument character limitation.
+     *
+     * @param root the directory to recursively search for JAR files.
+     * @throws java.net.MalformedURLException If a provided JAR file URL is malformed
+     */
+    private static void addJarFileUrls(File root) throws Exception {
+        File[] children = root.listFiles();
+        if (children == null) {
+            return;
+        }
+        for (File child : children) {
+            if (child.isFile() && child.canRead() &&
+                    child.getName().toLowerCase().endsWith(".jar") && !child.getName().contains("slf4j")) {
+                addPath(child.getPath());
+            }
+        }
+    }
+
+    private static void addPath(String s) throws Exception {
+        File f = new File(s);
+        URL u = f.toURL();
+        URLClassLoader urlClassLoader = (URLClassLoader) ClassLoader.getSystemClassLoader();
+        Class<URLClassLoader> urlClass = URLClassLoader.class;
+        Method method = urlClass.getDeclaredMethod("addURL", URL.class);
+        method.setAccessible(true);
+        method.invoke(urlClassLoader, u);
```

Comment:
This is bit dangerous. Correct way to create a new classloader and use it to execute the next instructions. 

@nandika  Please create new issue(improvement) explaining the above. I will approve this as the code works for now.